markdown/3.x/en/ff-middleware.md

```diff
@@ -22,7 +22,8 @@ Be sure to make `ff-middleware` an immediate child node of `ff-communication` li
 In order to configure middleware using JavaScript you attach an event handler to the `ffReady` event. Modules are registered by calling `use()` of the desired middleware namespace.
 
 ```javascript
-window.addEventListener("ffReady", () => {
+window.addEventListener("ffReady", (event) => {
```

Comment:
Arrow function is not supported by IE. Make sure there's a note about it before using in example. Remove braces - we don't use them for single argument.